core/src/Instance/AadAuthority.cs

```diff
@@ -52,6 +52,8 @@ internal class AadAuthority : Authority
             "login.cloudgovapi.us" // Microsoft Azure US Government
         };
 
+        internal static string B2CTrustedHost = "b2clogin.com";
```

Comment:
> internal static string B2CTrustedHost = "b2clogin.com"; [](start = 7, length = 56)

ideally we would have this in the B2CAuthotity class and then we have a virtual method which we override there which does the B2Clogin addition to the validation.